app.js

```diff
@@ -0,0 +1,174 @@
+var notesList = document.querySelector(".notes-list");
+var noteTitle = document.querySelector(".note-title");
+var notenote = document.querySelector(".note-note");
+var noteItem = document.querySelector(".note-item");
+var notenote = document.querySelector(".note-note");
+var emptyState = document.querySelector(".empty-state");
+
+var selectedNote = 0;
+var notes = [];
+
+toggleSwitch = document.querySelector('.theme-switch input[type="checkbox"]');
+
+function switchTheme(e) {
+  if (e.target.checked) {
+    document.documentElement.setAttribute("data-theme", "dark");
+    localStorage.setItem("theme", "dark");
+  } else {
+    document.documentElement.setAttribute("data-theme", "light");
+    localStorage.setItem("theme", "light");
+  }
+}
+
+if (currentTheme) {
+  toggleSwitch.checked = true;
+}
+
+toggleSwitch.addEventListener("change", switchTheme, false);
+
+function initializeNotes() {
+  var notesArr = localStorage.getItem(["notes"]);
+  if (JSON.parse(notesArr) == null || JSON.parse(notesArr).length === 0) {
+    notes = [
+      {
+        title: "Welcome to NotesHD!",
+        note: `This is a sample note, please feel free to edit or delete this note and get started with creating your own notes!`,
+      },
+    ];
+  } else {
+    notes = JSON.parse(notesArr);
+  }
+}
+
+function getNotesCount() {
+  return Number(notes.length);
+}
+
+function isLastEmpty() {
+  if (
+    notes[getNotesCount() - 1].title === "" &&
+    notes[getNotesCount() - 1].note === ""
+  ) {
+    return true;
+  }
+  return false;
+}
+
+function titleChangeHandler(idx) {
+  var noteTitle = document.querySelector(".note-title");
+  idx = Number(idx);
+  notes[idx].title = noteTitle.innerHTML;
+  // if (noteTitle.innerHTML === "") {
+  //   noteTitle.innerHTML = "Untitled";
+  // }
+  renderNotesList(false);
+}
+
+function noteChangeHandler(idx) {
+  var notenote = document.querySelector(".note-note");
+  idx = Number(idx);
+  notes[idx].note = notenote.value;
+  renderNotesList(false);
+}
+
+function addNote() {
+  var noOfNotes = notes.length;
+  if (noOfNotes != 0) {
```

Comment:
the check can be simplified to just `if(noOfNotes)` because 0 evaluates to false